pkg/executor/sortexec/sort.go

```diff
@@ -260,6 +266,7 @@ func (e *SortExec) Next(ctx context.Context, req *chunk.Chunk) error {
 	if e.fetched.CompareAndSwap(false, true) {
 		err := e.initCompareFuncs(e.Ctx().GetExprCtx().GetEvalCtx())
 		if err != nil {
+			e.closeChannels()
```

Comment:
When `e.fetchChunks` return error, don't need to close channels explicitly?

Reply:
> When `e.fetchChunks` return error, don't need to close channels explicitly?

Yes, because these channels have been closed in the `e.fetchChunks`.